backend/tests/test_cinq_auditor_aws_required_tags_rds.py

```diff
@@ -0,0 +1,133 @@
+import datetime
+import random
+import uuid
+
+from cloud_inquisitor.constants import AuditActions
+from cloud_inquisitor.plugins.types.resources import RDSInstance
+from tests.libs.cinq_test_cls import MockRequiredTagsAuditor
+from tests.libs.lib_cinq_auditor_aws_required_tags import prep_rds_testing, VALID_TAGS, STANDARD_ALERT_SETTINGS_STOP
+from tests.libs.util_cinq import setup_test_aws
+from tests.libs.util_db import create_resource
+
+
+def test_rds_generic(cinq_test_service):
+    """
+    Test will pass if for an S3 bucket meet the following condition:
```

Comment:
Can you update the docstring to reflect the RDS tests (I'm guessing this was a paste from S3 test)

Reply:
Fixed. Thanks!